tools/coverage/jacoco-unified.gradle

```diff
@@ -0,0 +1,72 @@
+apply from: "$rootDir/tools/coverage/jacoco-common.gradle"

```

Comment:
It would be nice if you could provide some comments explaining what it's happening on each code block, as there's a lot going on here and it's all inside one block.
It would be even better if you could split some stuff into separated methods with meaningful names. 🤗